### PR TITLE
[importer] Extend mapping capabilities. 

### PR DESCRIPTION
**Pull request by trasc**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
[importer] Extend mapping capabilities. 
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: cmd/importer/README.md [hunk 1]
```diff
@@ -20,14 +20,47 @@ go build  -C cmd/importer/ -o $(pwd)/bin/importer
...
+  toLocalQueue: user-queue2
+- match:
+    labels:
+      src.lbl: src-val3
+  skip: true
+```
+
+- During the mapping, if the match rule has no `priorityClassName` the `priorityClassName` of the pod will be ignored, if more than one `label: value` pairs are provided, all of them should match.
+- The rules are evaluated in order.
+- `skip: true` can be used to ignore the pods matching a rule.
```
Comment (alculquicondor) [inline cmd/importer/README.md:61]: could the user leave a rule with no priorityClassName and no labels with `skip:true`?

That could be a way of saying: if it doesn't match any rule above, I just don't care about this pod.

File: cmd/importer/pod/check.go [hunk 2]
```diff
@@ -32,40 +32,45 @@ import (
...
 		rf, rfFound := cache.ResourceFalvors[rfName]
 		if !rfFound {
-			return fmt.Errorf("%q flavor %q: %w", cq.Name, rfName, util.ErrCQInvalid)
+			return false, fmt.Errorf("%q flavor %q: %w", cq.Name, rfName, util.ErrCQInvalid)
+		}
+
+		var pv int32
+		if pc, found := cache.PriorityClasses[p.Spec.PriorityClassName]; found {
+			pv = pc.Value
 		}
```
Comment (alculquicondor) [inline cmd/importer/pod/check.go:66]: error if the `PriorityClassName!=""` and it is not found

File: cmd/importer/pod/check.go [hunk 3]
```diff
@@ -32,40 +32,45 @@ import (
...
 
-		log.V(2).Info("Successfully checked", "pod", klog.KObj(p), "clusterQueue", klog.KObj(cq), "resourceFalvor", klog.KObj(rf))
-		return nil
+		log.V(2).Info("Successfully checked", "clusterQueue", klog.KObj(cq), "resourceFalvor", klog.KObj(rf), "priority", pv)
```
Comment (alculquicondor) [inline cmd/importer/pod/check.go:68]:
```suggestion
		log.V(2).Info("Successfully checked", "clusterQueue", klog.KObj(cq), "resourceFlavor", klog.KObj(rf), "priority", pv)
```

File: cmd/importer/README.md [hunk 4]
```diff
@@ -20,14 +20,47 @@ go build  -C cmd/importer/ -o $(pwd)/bin/importer
...
 
 - At least one `namespace` is provided.
-- The label key (`queuelabel`) providing the queue mapping is provided.
-- A mapping from one of the encountered `queuelabel` values to an existing LocalQueue exists.
+- For every Pod a  mapping to a LocalQueue is available.
+- The target LocalQueue exists.
 - The LocalQueues involved in the import are using an existing ClusterQueue.
 - The ClusterQueues involved have at least one ResourceGroup using an existing ResourceFlavor. This ResourceFlavor is used when the importer creates the admission for the created workloads.
 
+The are two ways the mapping from a pod to a local queue can be specified:
```
[the remark this answers is not in the document]
Comment (trasc) [reply cmd/importer/README.md]: done

File: cmd/importer/README.md (hunk 1 above)
Comment (trasc) [reply cmd/importer/README.md:61]: yes,

```yaml
-skip: true
```

at the end is doing that.

similarly 
```yaml
-toLocalQueue: user-queue2
```
will put all the pods in one localQueue

File: cmd/importer/pod/check.go (hunk 2 above)
Comment (trasc) [reply cmd/importer/pod/check.go:70]: done

File: cmd/importer/pod/check.go (hunk 3 above)
Comment (trasc) [reply cmd/importer/pod/check.go:72]: done

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

File: cmd/importer/README.md [hunk 5]
```diff
@@ -36,7 +69,31 @@ After which, if `--dry-run=false` was specified, for each selected Pod the impor
...
+- match:
+    priorityClassName: p-class
+    labels:
+      src.lbl: src-val2
+      src2.lbl: src2-val
+  toLocalQueue: user-queue2
+```
+
+```bash
+./bin/importer import -n ns1,ns2  --queuemapping-file=<mapping-file-path> --dry-run=false
```
Comment (yaroslava-serdiuk) [inline cmd/importer/README.md:95]: Could script return `done` or smth if the import finished successfully?

**Pull request merged**